frontend/src/pages/CalendarPage/index.js

```diff
@@ -205,6 +218,14 @@ class CalendarPage extends React.Component {
                 )}
               </Paper>
             </Grid>
+<<<<<<< HEAD
```

Comment:
There's a merge conflict here, can you fix that? It's breaking your deploy previews